bliss/catalog.py

```diff
@@ -5,11 +5,13 @@
 from typing import Dict, Tuple
 
 import torch
+import torch.nn.functional as F
 from astropy import units as u  # noqa: WPS347
 from astropy.table import Table, hstack
 from astropy.wcs import WCS
 from einops import rearrange, reduce, repeat
 from torch import Tensor
+from torch.nn.utils.rnn import pad_sequence
```

Comment:
Could we use the regular pad function instead, even if it's a bit more verbose? Future developers may not be familiar with the rnn submodule

Reply:
I failed to find a better equivalent of torch's `pad_sequence`. I search for such function using gpt, and it gives the following result:
```python
def pad_sequence(sequences, padding_value=0):
    max_len = max(len(seq) for seq in sequences)
    padded_seqs = [
        seq + [padding_value] * (max_len - len(seq))
        for seq in sequences
    ]
    return padded_seqs

# Example usage
sequences = [[1, 2, 3], [4, 5], [6]]
padded_seqs = pad_sequence(sequences)
print(padded_seqs)
```

Maybe we can write more comments for `pad_sequence` to explain it better.

Reply:
OK I see, I'm onboard with using the rnn submodule.